release/models/bgp/openconfig-bgp-neighbor.yang

```diff
@@ -197,7 +203,7 @@ submodule openconfig-bgp-neighbor {
         type oc-inet:port-number;
         default 179;
         description
-          "Port number of the BGP peer";
+          "Destination TCP port number of the BGP peer";
```

Comment:
"Destination TCP port number of the BGP peer **when initiating a session from the local router**"; ?

In a BGP session, my understanding is that both sides need to configure each other as neighbours, and as result both sides have `neighbor-port` configured (either explicitly or via the default value). `neighbor-port` can therefore only be different from `remote-port` if the connection was initiated by the neighbour via a different port number. When this happens we are ignoring this configuration.